resources/hg-ReferralServiceRequest.xml

```diff
@@ -1,273 +1,277 @@
-        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthcareProvider-Organization" />
-      </type>
-      <mapping>
-        <identity value="ART-DECOR-hg-Envelop" />
-        <map value="hg-dataelement-4989" />
-        <comment value="Zorgaanbieder" />
-      </mapping>
-    </element>
-    <element id="ServiceRequest.supportingInfo">
-      <path value="ServiceRequest.supportingInfo" />
-      <slicing>
-        <discriminator>
-          <type value="profile" />
-          <path value="resolve()" />
-        </discriminator>
-        <rules value="open" />
-      </slicing>
-      <min value="1" />
-    </element>
-    <element id="ServiceRequest.supportingInfo:referralComposition">
-      <path value="ServiceRequest.supportingInfo" />
-      <sliceName value="referralComposition" />
-      <min value="1" />
-      <max value="1" />
-      <type>
-        <code value="Reference" />
-        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/hg-ReferralComposition" />
-      </type>
-      <mapping>
-        <identity value="ART-DECOR-hg-Envelop" />
-        <map value="hg-dataelement-1673" />
-        <comment value="Envelop" />
-      </mapping>
-      <mapping>
+        <uri value="https://decor.nictiz.nl/pub/eerstelijnszorg/hg-html-20221201T095845/ds-2.16.840.1.113883.2.4.3.11.60.103.1.1-2020-10-19T175239.html" />
+        <name value="Huisarts- en paramedische overdrachten - Envelop" />
+        <comment value="This is a preliminary mapping that should not be used for immediate implementation." />
+    </mapping>
+    <mapping>
         <identity value="ART-DECOR-hg-Kern" />
-        <map value="hg-dataelement-1709" />
-        <comment value="Kern" />
-      </mapping>
-    </element>
-  </differential>
+        <uri value="https://decor.nictiz.nl/pub/eerstelijnszorg/hg-html-20221201T095845/ds-2.16.840.1.113883.2.4.3.11.60.103.1.1-2020-10-19T175239.html" />
+        <name value="Huisarts- en paramedische overdrachten - Kern" />
+        <comment value="This is a preliminary mapping that should not be used for immediate implementation." />
+    </mapping>
+    <kind value="resource" />
+    <abstract value="false" />
+    <type value="ServiceRequest" />
+    <baseDefinition value="http://hl7.org/fhir/StructureDefinition/ServiceRequest" />
+    <derivation value="constraint" />
+    <differential>
+        <element id="ServiceRequest">
+            <path value="ServiceRequest" />
+            <mapping>
+                <identity value="ART-DECOR-hg-Envelop" />
+                <map value="hg-dataelement-1673" />
+                <comment value="Envelop" />
+            </mapping>
+        </element>
+        <element id="ServiceRequest.status">
+            <path value="ServiceRequest.status" />
+            <definition value="In current referral use cases the referral is considered done when sent by the referral initiator. Therefore, status is set to a fixed value of _completed_." />
```

Comment:
Reading this literally, it seems like we have given `.status` a fixed value of _completed_ in the profile itself (which isn't the case, because this probably isn't desired in the near future when other use cases are supported as well). Shouldn't we formulate it then as 'Therefore, in those use cases the status is always set to _completed_.' (and you could consider to use '`.status`' instead of 'status').

Reply:
@LuudSlagter You are right. I will change this.